scheduler/replica_scheduler.go

```diff
@@ -365,6 +360,41 @@ func (rcs *ReplicaScheduler) filterNodeDisksForReplica(node *longhorn.Node, disk
 	return preferredDisks, multiError
 }
 
+// filterDisksWithMatchingReplicas filters the input disks map and returns only the disks that have the fewest matching
+// replicas. If allowDuplicates is false, it only returns disks that have no matching replicas.
```

Comment:
Did the parameter name change from allowDuplicates to allowMatches?

Reply:
Fixing.